framework/resources/js/query.js

```diff
@@ -614,8 +614,12 @@
 						callback()
 					}
 					
-					$(document).trigger('fw_query_success', [item])
-					
+					$(document).trigger('fw_query_success', [item]);
+
+					// Scroll to the top of the results.
+					const scroll_top = $('.query-page').position().top;
+
+					$( 'html, body' ).animate( {scrollTop: scroll_top}, 10, 'swing' );
```

Comment:
The scrolling effect should not be inside this function, because the function is used in different places, and we could desire a different effect in the different places. For example, in the Maps page, we use this plugin, but we don't want to scroll the whole page, only a specific container.

Suggestion: in the child-functions.js script, there are page specific scripts. A new function could be created there to listen to the `fw_query_success` event. But also open to do the effect somewhere else.